src/mame/drivers/megaplay.cpp

```diff
@@ -628,7 +628,7 @@ uint32_t mplay_state::screen_update_megplay(screen_device &screen, bitmap_rgb32
 	screen_update_megadriv(screen, bitmap, cliprect);
 	//m_vdp1->screen_update(screen, bitmap, cliprect);
 
-	// i'm not sure if the overlay (256 pixels wide) is meant to be stretched over the 320 resolution genesis output, or centered.
+	// TODO : the overlay (256 pixels wide) is actually stretched over the 320 resolution genesis output, reference is https://youtu.be/Oir1Wp6yOq0.
```

Comment:
This is called superimposition effect, something that MAME barely supports for laserdisc games.